### DISCUSSION TITLE
filter default value if empty

### DISCUSSION THREAD

**Discussion by pgswow** [pocketbase/pocketbase#1131] [Q&A] [answered]
Hello!
can i use some default value for the filter or empty value which means don't apply the filter or output all values if the variable is not set?

**Comment by ganigeorgiev** [comment, marked as answer]
What do you mean by "filter default value if empty"?

When performing list/search requests (aka. `getList()`), the `filter` query param is optional and it will return all records that satisfy the collection's "List API rule".

**Reply by pgswow** [reply]
I think that I did not formulate the question correctly, and in general I think the question is meaningless =) I'm sorry =)
